tools/pot/openvino/tools/pot/graph/graph_utils.py

```diff
@@ -9,7 +9,7 @@
 from openvino.tools.mo.utils.logger import init_logger
 from openvino.runtime import Core  # pylint: disable=E0401,E0611
 from openvino.runtime.passes import Manager # pylint: disable=E0401,E0611
-from openvino.offline_transformations_pybind import apply_pot_transformations # pylint: disable=import-error,no-name-in-module
+from openvino.offline_transformations import apply_pot_transformations # pylint: disable=import-error,no-name-in-module
```

Comment:
Please add try-except block with the old and new import to avoid POT import errors.

Reply:
done